src/cmds/pbs_attach.c

```diff
@@ -397,6 +372,34 @@ main(int argc, char *argv[])
 		 */
 		(void)putenv("MPICH_PROCESS_GROUP=no");
 
+		/*
+		 ** Load in the job environment variables needed for cgroups
+		 ** Check to see the file $PBS_NODEFILE.env exists
+		 */
+		snprintf(jobenvfile, sizeof(jobenvfile), "%s/mom_priv/jobs/%s.env", pbs_conf.pbs_home_path, jobid);
+		if ((fp = fopen(jobenvfile, "r"))) {
+			while(fgets(line, sizeof(line), fp)) {
+				p = line + strlen(line);
+				while (--p >= line) {
+					if (!(*p == '\n' || *p == '\r' || *p == ' ' || *p == '\t'))
+						break;
+					*p = '\0';
+				}
+				if (strlen(line) > 0) {
+					p = strdup(line);
+					if (p) {
+						(void) putenv(p);
+						i++;
+					}
+				}
+			}
+			fclose(fp);
+			fprintf(stderr, "%s: PBS_JOB_ENV_FILE contains %d entries", jobid, i);
```

Comment:
The filename is not PBS_JOB_ENV_FILE - so maybe you can print the actual filename itself